Makefile

```diff
@@ -174,10 +174,26 @@ go.cachedir:
 # This is for running out-of-cluster locally, and is for convenience. Running
 # this make target will print out the command which was used. For more control,
 # try running the binary directly with different arguments.
+
+CROSSPLANE_NAMESPACE ?= crossplane-system
+TLS_SERVER_CERTS_DIR ?= ./.cache/tls/server
+TLS_CLIENT_CERTS_DIR ?= ./.cache/tls/client
+CROSSPLANE_CACHE_DIR ?= ./.cache/crossplane
 run: go.build
+	@$(INFO) Getting TLS certificates from the cluster...
+	@mkdir -p $(TLS_SERVER_CERTS_DIR)
+	@kubectl get secret crossplane-tls-server -n $(CROSSPLANE_NAMESPACE) -o jsonpath="{.data.tls\.crt}" | base64 --decode > "$(TLS_SERVER_CERTS_DIR)/tls.crt"
```

Comment:
instead of directly invoking `kubectl`, I think you'd want to use `$(KUBECTL)` instead, so the cached tool/version is used instead of whatever is on your local machine 😉

Reply:
good catch 💯